src/test/java/com/snowflake/kafka/connector/streaming/iceberg/IcebergIngestionSchemaEvolutionIT.java

```diff
@@ -138,4 +110,185 @@ private static Stream<Arguments> prepareData() {
+    insertWithRetry(testStruct5, 4, false);
+    waitForOffset(5);
+
+    columns = describeTable(tableName);
+    assertEquals(
+        columns.get(1).getType(),
+        "OBJECT(k1 NUMBER(19,0), k2 NUMBER(19,0), k3 VARCHAR(16777216), k4 FLOAT, k5 NUMBER(19,0),"
+            + " k6 NUMBER(19,0))");
+    assertEquals(columns.size(), 2);
+  }
+
+  @Test
+  public void alterAlreadyExistingStructure_timestamp() throws Exception {
+    String testStruct1 =
+        "{"
+            + "    \"_id\": \"673f3b93a56dd01a8a0cb6a4\","
+            + "    \"index\": 0,"
+            + "    \"guid\": \"738142bb-5878-42ad-bf35-6015f63b67dd\","
+            + "    \"isActive\": true,"
+            + "    \"balance\": \"$1,690.88\","
+            + "    \"picture\": \"http://placehold.it/32x32\","
+            + "    \"age\": 38,"
+            + "    \"eyeColor\": \"blue\","
+            + "    \"name\": \"Davis Heath\","
+            + "    \"gender\": \"male\","
+            + "    \"company\": \"EVENTEX\","
+            + "    \"email\": \"davisheath@eventex.com\","
+            + "    \"phone\": \"+1 (987) 471-3852\","
+            + "    \"address\": \"768 Cypress Court, Lookingglass, Kansas, 5659\","
+            + "    \"about\": \"Nisi voluptate id occaecat nisi pariatur dolore laborum labore ea"
+            + " reprehenderit consequat sint fugiat sunt. Et consequat esse ex cillum deserunt"
+            + " Lorem. Enim nisi tempor non nisi. Consectetur ut ad reprehenderit fugiat et"
+            + " adipisicing sint. Deserunt est proident exercitation sit cillum in non excepteur"
+            + " aliqua qui amet cillum sint aliquip.\\r"
+            + "\","
+            + "    \"registered\": \"2023-09-19T09:41:45 -02:00\","
+            + "    \"latitude\": 13.997901,"
+            + "    \"longitude\": 130.854106,"
+            + "    \"tags\": ["
+            + "      \"et\","
+            + "      \"ut\","
+            + "      \"elit\","
+            + "      \"do\","
+            + "      \"nostrud\","
+            + "      \"id\","
+            + "      \"veniam\""
+            + "    ],"
+            + "    \"friends\": ["
+            + "      {"
+            + "        \"id\": 0,"
+            + "        \"name\": \"Sandoval Hodges\""
+            + "      },"
+            + "      {"
+            + "        \"id\": 1,"
+            + "        \"name\": \"Ramirez Brooks\""
+            + "      },"
+            + "      {"
+            + "        \"id\": 2,"
+            + "        \"name\": \"Vivian Whitfield\""
+            + "      }"
+            + "    ],"
+            + "    \"greeting\": \"Hello, Davis Heath! You have 4 unread messages.\","
+            + "    \"favoriteFruit\": \"strawberry\""
+            + "  }";
+    insertWithRetry(testStruct1, 0, false);
+    waitForOffset(1);
+
+    List<DescribeTableRow> columns = describeTable(tableName);
+    assertEquals(columns.size(), 23);
```

Comment:
what we are checking here? Some magic numbers to me. Do we check some field was added or all of the fields? We should run also describeTable before schema evolution to compare